### ISSUE TITLE
Question concerning the animation of the Rubik cube under povray

### ISSUE THREAD

**Issue by oineant** [EVGVir/rubik-cube#1]
Hi,

Is it possible to animate the cube using the notations f, b, r, l, u, p ? 
These notations allow the rotation of two layers  at the same time, for example: r = R + M'

Thank you in advance

**Comment by EVGVir** [comment]
No, there is no such possibility. Moreover, unfortunately, it does not look like a simple task to add this functionality. Because current architecture works with one layer at a time.

By the way, do you need this notation for a cube of which size?

**Comment by oineant** [comment]
I thank you for your prompt reply.

I need these notations for a 3x3x3 cube. 
For example: f R U R’ U’ f’, we rotate at the same time F & S (using f) then F' & S' (using f').

In your file rubik-cube-notation.inc, you included David Singmaster's notation plus the extended ones M, E, S so, I thought that's possible to animate both layers at the same time.

I'm using your rubik-cube package to animate the cube. It's for educational purposes. I'm finishing the 1-look PLL algorithms and I'll soon start the 1-Look OLL ones which contain a lot of these notations (obviously I'm going to get stuck).

Here's an example of H perm (PLL): The cube is at the center of the screen, then two mirrors are added (one to the left and another one to the right) showing the right side and the left side of the cube while showing at the same time its back side. I included a top view of the cube configuration and the algorithm solving it (the animation starts after five seconds and stops at the end for 5 seconds).  

https://user-images.githubusercontent.com/96337575/146663211-b9c47410-6915-40f4-a61e-ff3885f83213.mp4

Thanks again.
